contrib/test.sh

```diff
@@ -41,3 +41,11 @@ then
         cargo test --features integration
     )
 fi
+# Lint if told to
+if [ "$DO_LINT" = true ]
+then
+    (
+        rustup component add rustfmt
```

Comment:
it looks it is already installed, CI prints:
`info: component 'rustfmt' for target 'x86_64-unknown-linux-gnu' is up to date`